### ISSUE TITLE
Picture of PCB?

### ISSUE THREAD

**Issue by Insood** [directive0/Digimon-Maturation#1]
Hi! I really enjoyed your article/video/code for the digimon v-pet hack. I'm looking into seeing if it's possible to emulate the device. One thing I'd like to know is if the PCB has all of the important components blob'd. Can you take a picture of the internal PCB? I only caught a view of the rear which didn't look like it had many components on it.

**Comment by directive0** [comment]
Hey! That's a really cool idea that I see having some neat applications!

I'd be happy to get you a picture of the Digimon which is now dead as a result of my fiddling with some power rails without thinking. Unfortunately I think most of what you'd need to construct a good software facsimile of the electronics is contained within the resin blob. However, if its helpful to you i'd be happy to send you the PCB and you can try decapsulating it on your own?

I'd love to help in any way I can. Let me know.

**Comment by Insood** [comment]
Ah bummer. I'd like to try to decapping it though! Can I toss you a couple bucks for shipping the PCB? Thanks!

**Comment by directive0** [comment]
Yeah sure, thats no problem, but can we agree that whatever you find you'll share it with the community so that we can all benefit? I'm sure that was probably your intention but just wanted to be explicit about it. I'll probably send it by whatever is the cheapest and easiest. 

Whats your address? It'll be coming from Toronto Canada. Let me find out what shipping will cost and I'll let you know if I need reimbursement.

What sort of platform would you want to write the emulator in? I'm really not familiar with any of that stuff so I'm just wondering. Ideally it would be amazing if we could figure out how the Digimons handle their communication too. But I'm getting a head of myself. Let me know your addy when you can. I'll be gone for two weeks starting this monday so if I can figure out how to send it before then great.

**Comment by Insood** [comment]
Yeah - no problem, I'm definitely going to be documenting and releasing everything. 

Actually - don't worry about sending anything just yet. I'm getting two coming in next week from Ebay so hopefully I can make some small amount of progress from those two. 

As for the emulator - probably C++ or Go. I think the communication protocol is already well documented though. I haven't toyed around with it yet, but [here's](http://s000.tinyupload.com/index.php?file_id=05238582279288663556) a link to the code and a circuit diagram if you want to fiddle around with it.

**Comment by directive0** [comment]
Hey, how goes it? I'm thinking of gearing this project up for V2. Would love to hear about any progress you've made!

I still have that board if you want it.

**Comment by Insood** [comment]
Eh, it's stalled out quite a bit. [Here](https://withthewill.net/threads/18065-Picture-of-a-Digimon-V-Pet-PCB/page2?p=318358&viewfull=1#post318358)'s everything I found out about it.
